filters.json.example

```diff
@@ -426,11 +426,13 @@
     },
     "eggs":{
         "enabled":"True",
+        "gymname_contains":["sprint","level badge","phone skin","starbucks"],
```

Comment:
I feel like this is better as `"contains"` but I'm interested to hearing why you decided on gymname_contains.

Reply:
I used gymname_contains because it was crystal clear.  'Contains' by itself could mean other things such as a pokemon in the gym or a trainer in the gym or the type of pokemon for the raid or the street address or some other attribute.  I'd prefer something more succinct but I erred on the side of specificity in the code that was user-visible.  I think someone looking at a JSON that has this section filled out will know what 'contains' means but otherwise it requires looking at the wiki to be sure.